### PR TITLE
Refresh README instructions

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Some small improvements with changes we've made over time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,33 @@ API docs authors can preview their changes to one of the APIs by using the `-a`
 1. Run `npm run gen-api -- -p <pkg-name> -v <version> -a <path/to/docs/_build/html>`.
 2. Execute `./start` and open up `http://localhost:3000`, as explained in the prior section.
 
-## Execute notebooks
+## Jupyter notebooks
+
+### Add a new notebook
```
Comment (Eric-Arellano) [inline README.md:94]: This section was moved up from below. No edits made.

File: README.md [hunk 2]
```diff
@@ -440,10 +440,48 @@ Next, choose the file name. The file name will determine the URL. For example, `
...
+```
+---
+title: Representing quantum computers
+description: Learn about coupling maps, basis gates, and backend errors for transpiling
+---
+```
+
+In Jupyter notebooks, set `title` and `description` in the `metadata` section for the file. In VSCode, you can set up the metadata with these instructions:
+
+1. Open up the file with the "Open With..." option and then "Text Editor".
```
Comment (abbycross) [inline README.md:463]:
```suggestion
1. Open up the file with the "Open With..." option (one way to do this is to right-click the file name to find the "Open With..." option) and then "Text Editor".
```

File: README.md [hunk 3]
```diff
@@ -440,10 +440,48 @@ Next, choose the file name. The file name will determine the URL. For example, `
...
+2. Scroll down to the bottom of the file for the top-level key "metadata". Be careful that this is the metadata for the entire file and not a single code block. You should see in the "metadata" section other entries like "language_info" and "nbconvert_exporter".
```
Comment (abbycross) [inline README.md:464]:
```suggestion
2. Scroll down to the bottom of the file for the top-level key "metadata". Ensure that this is the metadata for the entire file and not for a single code block. You should see in the "metadata" section other entries like "language_info" and "nbconvert_exporter".
```

**Review by abbycross — APPROVED** [review]
a couple suggested tweaks - thanks, these are very useful updates!

**Pull request merged**